src/components/EmojiItem/hooks/useEmojiItem.ts

```diff
@@ -21,7 +21,7 @@ const useEmojiItem = (props: EmojiItemProps) => {
   const [xValue, setXValue] = useState<number>(0);
   const [titlePosition, setTitlePosition] = useState<number>(0);
   const scaled: boolean =
-    currentPosition > xValue && currentPosition < xValue + 30;
+    currentPosition > xValue && currentPosition < xValue + 20;
```

Comment:
Why decrease the size of 30 -> 20? Any reason behind that? And Yes Please add a description above the variable as well if it's possible.

Reply:
when I am giving the value 30 then for the first time when we start gesture all five emojis get scaled at the same time and giving unexpected behavior